contrib/skopeoimage/README.md

```diff
@@ -11,7 +11,7 @@
 [comment]: <> ()
 [comment]: <> (***ATTENTION*** ***WARNING*** ***ALERT*** ***CAUTION*** ***DANGER***)
```

Comment:
^^^

Tom, will you handle the Quay.io updates as well?

(Like this one, it’s just an URL component visible only to maintainers.)